src/main/java/org/cyclops/integrateddynamics/core/network/EnergyChannel.java

```diff
@@ -0,0 +1,90 @@
+package org.cyclops.integrateddynamics.core.network;
+
+import org.cyclops.integrateddynamics.GeneralConfig;
+import org.cyclops.integrateddynamics.api.network.IChanneledNetwork;
+import org.cyclops.integrateddynamics.api.network.IPositionedAddonsNetwork.PrioritizedPartPos;
+
+import net.minecraftforge.energy.IEnergyStorage;
+
+public class EnergyChannel implements IEnergyStorage {
+    final EnergyNetwork network;
```

Comment:
I tend to make these things private by default, any reason why you are not doing that? (Just a question, you shouldn't change this if you have a good reason)